packages/mdc-select/component.ts

```diff
@@ -399,17 +379,19 @@ export class MDCSelect extends MDCComponent<MDCSelectFoundation> {
         const menuItem = this.menu_.items[index];
         return menuItem && menuItem.textContent ? menuItem.textContent : '';
       },
-      toggleClassAtIndex: (index: number, className: string, toggle: boolean) => {
+      addClassAtIndex: (index: number, className: string) => {
         const menuItem = this.menu_.items[index];
```

Comment:
Foundation should guarantee to send valid index, again component does not need to check for correctness of input to adapter. ie, `menu_.items[index]` always returns valid menu item.